src/raftstore/store/fsm/store.rs

```diff
@@ -467,6 +468,22 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             warn!("set thread priority for raftstore failed, error: {:?}", e);
         }
 
+        {
+            // entry cache gc thread
+            // During stress test, on_raft_gc_log_tick may costs tens of seconds. This thread is
+            // used to reclaim memory of entry cache.
+            let (tx, rx) = mpsc::channel();
+            self.entry_cache_gc_handler.entry_gc_sender = Some(tx);
+            thread::Builder::new()
```

Comment:
Can we reuse the `raft_log_gc` thread to do the job?

Reply:
Worker use mutex and poll to handle task, which will bring in unnecessary overhead.

Reply:
Good idea!!!

Reply:
But i think there are too many threads in TiKV, we should be careful when we add a new thread.

Reply:
@zhangjinpeng1987 PTAL.
I have used `raftlog_gc_worker` to schedule entry gc task and also changed the structure of `Task`.